recipes/pathy/patches/0001-setup-requirements.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/setup.py b/setup.py
+index 0cc504e..7b66995 100644
+--- a/setup.py
++++ b/setup.py
+@@ -11,9 +11,11 @@ def setup_package():
+         about = {}
+         exec(f.read(), about)
+ 
+-    requirements_path = root / "requirements.txt"
```

Comment:
I guess this is because the `requirements.txt` file is missing from the sdist. I'd prefer a patch which re-adds it to make sure the metadata doesn't get out of sync when this is updated.

Also can you open an issue or PR for this upstream if you haven't already?

Reply:
@justindujardin is aware and plans to fix the sdist for the next release, but there's not a PR at this point. I will update the patch with the file instead.